contracts/utils/CurvedInterestRate.sol

```diff
@@ -1,51 +1,68 @@
-// SPDX-License-Identifier: Unlicense
+/// SPDX-License-Identifier: Unlicense
 pragma solidity >=0.8.4;
 
 import "prb-math/contracts/PRBMathUD60x18.sol";
 import "../interfaces/ICurvedInterestRate.sol";
+import "./Guardian.sol";
```

Comment:
utilize PausableGuardian_0_8.sol instead of making a new contract. this is found in the governance folder.

Reply:
fixed